scootapi/client.go

```diff
@@ -0,0 +1,101 @@
+package scootapi
```

Comment:
so i put it in scootapi because our /client package is really a command line tool?  Its more like CLI.  The idea was this is an actual go client for interacting with scootapi.  The thought being if you are looking up the thrift then you'd discover this client and use it hopefully.